packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts

```diff
@@ -275,3 +264,81 @@ describe('listAccountHistory', () => {
     expect(history.find((h) => h.type === 'AccountToUtxos' && h.amounts[0] === '-3.97000000@DFI')).toBeTruthy()
   })
 })
+
+describe('listAccountHistory for poolpair', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+  const createToken = createTokenForContainer(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await container.waitForWalletBalanceGTE(200)
+    const address = await container.call('getnewaddress')
+    await container.call('utxostoaccount', [{ [address]: '100@0' }])
+    await createToken(address, 'DDAI', 2000)
+
+    await client.poolpair.createPoolPair({
+      tokenA: 'DFI',
+      tokenB: 'DDAI',
+      commission: 0,
+      status: true,
+      ownerAddress: await container.call('getnewaddress')
+    })
+    await container.generate(1)
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should show AddPoolLiquidity', async () => {
+    await client.poolpair.addPoolLiquidity({
+      '*': ['10@DFI', '200@DDAI']
+    }, await container.call('getnewaddress'))
+    await container.generate(1)
+
+    const histories = await client.account.listAccountHistory()
+
+    expect(histories.find((h) => (h.type === 'AddPoolLiquidity' && h.amounts.includes('-10.00000000@DFI') && h.amounts.includes('-200.00000000@DDAI')))).toBeTruthy()
```

Comment:
I think it will be better to write it line this

```suggestion
     const records = histories.filter((h) => (h.type === 'AddPoolLiquidity' 
     expect(records).size to be 1?
     expect(records[0].amounts.include))..
     expect(records[0].amounts.include))..
```

easier to read and see the structure

Even better would be using to strict equal with `expect.any` 

https://github.com/DeFiCh/whale/blob/d1d876fb8120d7101ed0eccd03f1b0084a601a70/packages/whale-api-client/__tests__/api/poolpair.test.ts#L77-L102

Reply:
There will be 2 `AddPoolLiquidity` records which its order is not guaranteed.

Those 2 records are:
1. Receiving `DFI-DDAI`
2. Sending out `DFI` and `DDAI`

Reply:
I've made changes to use a combination of `expect.arrayContaining` and `expect.objectContaining`.